commerce/domain/inventory/cart/calculator/classes/InventoryCartCalculatorSampleTest.cls

```diff
@@ -0,0 +1,199 @@
+    private static final String WEBSTORE_NAME = 'My WebStore';
+    private static final String DELIVERYGROUP_NAME = 'My Delivery Group';
+    private static final String CART_ITEM1_NAME = 'My Cart Item 1';
+    private static final String CART_ITEM2_NAME = 'My Cart Item 2';
+    private static final String CART_ITEM3_NAME = 'My Cart Item 3';
+    private static final String SKU1_NAME = 'My SKU 1';
+    private static final String SKU2_NAME = 'My SKU 2';
+    private static final String SKU3_NAME = 'My SKU 3';
+
+    @IsTest
+    public static void testNoCVOCartStatus() {
+      
+         // Arrange
+         CartExtension.Cart cart = arrangeAndLoadCartWithSpecifiedStatusAndThreeItems(CartExtension.CartStatusEnum.ACTIVE);
+         InventoryCartCalculatorSample calculator = new InventoryCartCalculatorSample(new DefaultInventoryCartCalculatoMockExecutor());
+
+        Test.startTest();
+        calculator.calculate(new CartExtension.CartCalculateCalculatorRequest(cart, CartExtension.OptionalBuyerActionDetails.empty()));
+        Test.stopTest();
+
+        // Assert
+        assertNoCartValidationOutputs(cart, 0);
+
+        
+    }
+    
+    @IsTest
+    public static void testCleanCVOCartStatus() {
+      
+         // Arrange
+         CartExtension.Cart cart = arrangeAndLoadCartWithSpecifiedStatusAndThreeItems(CartExtension.CartStatusEnum.ACTIVE);
+        
+         InventoryCartCalculatorSample calculator = new InventoryCartCalculatorSample(new DefaultInventoryCartCalculatoMockExecutor());
+
+        Test.startTest();
+        calculator.calculate(new CartExtension.CartCalculateCalculatorRequest(cart, CartExtension.OptionalBuyerActionDetails.empty()));
+        Test.stopTest();
+
+        // Assert
+        assertNoCartValidationOutputs(cart, 0);
+
+        
+    }
+
+     @IsTest
+    public static void testWithCVOCartStatus() {
+      
+         // Arrange
+         CartExtension.Cart cart = arrangeAndLoadCartWithSpecifiedStatusAndThreeItems(CartExtension.CartStatusEnum.CHECKOUT);
+         addCvoToCart(cart);
+         InventoryCartCalculatorSample calculator = new InventoryCartCalculatorSample(new DefaultInventoryCartCalculatoMockExecutor());
+
+        Test.startTest();
+        calculator.calculate(new CartExtension.CartCalculateCalculatorRequest(cart, CartExtension.OptionalBuyerActionDetails.empty()));
+        Test.stopTest();
+
+        // Assert
+        assertNoCartValidationOutputs(cart, 1);
+
+        
+    }
+  
+    
+     /**
+     * @description Sample mock executor. Stubs result of default pricing calculator
+     */
+    public class DefaultInventoryCartCalculatoMockExecutor extends CartExtension.CartCalculateExecutorMock {
+
+        /**
+         * @description This constructor should only be exposed to customers in a test context
+         */
+        public DefaultInventoryCartCalculatoMockExecutor() {}
+
+        public override void defaultInventory(CartExtension.CartCalculateCalculatorRequest request) {
```

Comment:
This Mock is to stub `super.calculate()` invocation from your Calculator class. I don't think you want to make any assumption about what this invocation is doing for the purpose of your tests? Is it better to just noop here and return immediately?